docs/projects/version-control/config-as-code-reference.md

```diff
@@ -213,17 +214,22 @@ connectivity_policy {
 versioning_strategy {
 
     donor_package {
-        step = "Deploy a Package"
+        step = "deploy-a-package"
     }
 }
 ```
 
+## Slugs in OCL
+
+Previously, version controlled projects would reference shared resources using their name. This had a side-effect causing API responses for version controlled projects to contained names instead of IDs.
+This has recently changed. Shared resources are now referenced from OCL by their slug. IDs will be used in API responses instead of names.
+
 ## Items of note
 
 When designing the config-as-code feature, we made several decisions to keep an appropriate balance of usability and functionality. There are a few limitations and items of note you should be aware of with config-as-code.
 
 - The Octopus Terraform Provider and OCL are not a 1:1 match. You cannot copy resources between the two and expect everything to work. We want to narrow the gap as much as possible, but as of right now, a gap exists.
-- Shared resources (environments, external feeds, step templates, etc.) are referenced by name, not ID. Be very careful when changing the names of any shared resources.  
+- Shared resources (environments, external feeds, step templates, etc.) are referenced by their slug, not ID.
```

Comment:
Should we also clarify here that it's by slug in OCL, but ID in the API?

Reply:
Good call, updated the wording here 👍